app/__init__.py

```
@@ -1,137 +1,126 @@
+
+try:
+    from app.schemas.bmi_v1 import BMIRequestV1  # type: ignore[no-redef]
+except ImportError:
+    pass
+
+# Routers
+from app import routers
+
+vip_router: Optional[Any] = None
+try:
+    from app.routers.vip import router as vip_router  # type: ignore[no-redef]
```

Flake8 (7.3.0)

[error] 'app.routers.vip.router as vip_router' imported but unused

(F401)